kde-frameworks/purpose-kaccounts-services/purpose-kaccounts-services-9999.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# this is purely for service file creation
+ECM_I18N="false"
+ECM_HANDBOOK="false"
+KDE_ORG_NAME="${PN/-kaccounts-services/}"
+KF5_BDEPEND=( "kde-apps/kaccounts-integration:5" )
+KF6_BDEPEND=( "kde-apps/kaccounts-integration:6" )
+inherit ecm-common frameworks.kde.org
+
+DESCRIPTION="KAccounts generated service files for nextcloud and google services"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+RDEPEND="
+	!<kde-frameworks/purpose-5.116.0-r2:5
+	!<kde-frameworks/purpose-6.5.0-r1:6
+"
+
+ecm-common-check_deps() {
+	return $(has_version -b "kde-apps/kaccounts-integration:6")
+}
+
+ecm-common_inject_heredoc() {
+	cat >> CMakeLists.txt <<- _EOF_ || die
```

Comment:
I actually worry a bit about new services being added here which we miss. Consider a sanity check? (Do find/glob and compare with a hardcoded list in the ebuild?)

Reply:
We could in this case do sth like `find $S -name "*service.in" | wc -l` and compare that with our currently known hardcoded value.

Reply:
That would be good enough.